essay.tex

```diff
@@ -0,0 +1,43 @@
+
+\section*{The Sprint}
+
+It is given that dominant forms of supervision best suit inexperienced team members who need direction whilst highly qualified staff will respond better to a Laissez Faire approach.  The analogy of managing a process instead of a person lends itself to the idea that perhaps dominant forms may suit a method where experience is not a factor and the Laissez Faire end of the spectrum where the team's individual expertise must be relied upon.  In which case, we must look at the different stages of a game development project or sprint in terms of freedom of direction and pressure to complete.  On the surface this would seem to be a direct relationship of pressure increasing proportionally over time, however, initially there must be a strong element of control.  In a game development cycle the start is often the planning stage where in lieu of absolute parameters the entire team is concerned with design and creativity.  A wide range of factors (e.g. artistic, emotive, and immersive factors) must be addressed by the requirements engineering effort \cite{callele2005requirements}.  Be it creating the assets, or scripting behaviour, care must be taken to keep the teams vision aligned with the project owner's objective without stifling the creativity of the individuals.  This may seem to warrant a soft touch, but in fact absolute instructions must be given.  The motivational level in respect of Laissez Faire is low because of not interference of management \cite{chaudhry2012impact}, creativity and experience must still be pointed in the required direction.
+
+Over the course of a sprint the need for control decreases.  The role of the scrum master mostly revolves around removing impediments to the process.  With user stories clearly assigned and in progress the Agile principle of trust must be adhered to.  Unless there are changing requirements to the projects parameters a soft touch can now be offered.  This is the point where capable staff thrive and as such, more demanding tasks should be delegated.  As the end of the sprint approaches there must be vigilant oversight to avoid overrunning or under achieving.  As deadlines near, unforeseen circumstances can have drastic implications and correctly identifying and managing these requires the autocratic strength of more dominant techniques. 
+
+\includegraphics{agile_graph.png}
```

Comment:
This graph is never referred to in the main body of the text. Please do this. Use the `figure` environment, and include a caption and reference. For details, refer to this [guide](https://en.wikibooks.org/wiki/LaTeX/Labels_and_Cross-referencing#Pictures).